cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerSetupService.java

```diff
@@ -323,10 +389,35 @@ public void publishPolicy(String template, boolean govCloud) {
                 apiConfigPolicy.setName("FISMA Policy For Login Banner");
 
                 List<ApiConfigEnforcement> apiConfigEnforcements = new ArrayList<>();
-                ApiConfigEnforcement apiConfigEnforcement = new ApiConfigEnforcement();
-                apiConfigEnforcement.setLabel("LOGIN_BANNER");
-                apiConfigEnforcement.setDefaultValue(banner);
-                apiConfigEnforcements.add(apiConfigEnforcement);
+
+                ApiConfigEnforcement loginBannerEnforcement = new ApiConfigEnforcement();
```

Comment:
Can we have a separate policy factory or policy configurer service for this Gov specific logic with UT?

Reply:
will do this. thanks Tomi